experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -124,7 +124,7 @@ function createResource(resource = {}): PerformanceResourceTiming {
 function createMainResource(resource = {}): PerformanceResourceTiming {
   const mainResource: any = createResource(resource);
   Object.keys(mainResource).forEach((key: string) => {
-    if (typeof mainResource[key] === 'number') {
+    if (typeof mainResource[key] === 'number' && mainResource[key] !== 0) {
       mainResource[key] = mainResource[key] + 30;
```

Comment:
I believe this is intended to "tweak" the timing but ends up affecting the body size, hence the changes to some of the expectations below. It's not great but #5130 and #5282 is actively working on unwinding these